conda_smithy/cli.py

```diff
@@ -228,9 +228,11 @@ def __call__(self, args):
                         )
                         print("")
                     else:
+                        quote = "'" if not sys.platform.startswith('win') else '"'
                         print(
                             "You can commit the changes with:\n\n"
-                            "    git commit -m 'MNT: Re-rendered with conda-smithy %s'\n" % __version__
+                            "    git commit -m {q}MNT: Re-rendered with conda-smithy {version}{q}\n"
```

Comment:
I only used `'` because the surrounding string is `"`, so I wouldn't have to escape. No deeper reason than that. Switching the inner and outer quotes should work fine, too, or escape the inner quotes.